src/app/loans/loans-routing.module.ts

```diff
@@ -5,113 +5,139 @@ import { Routes, RouterModule } from '@angular/router';
 /** Translation Imports */
```

Comment:
The loan module is lazy loaded with the clients component. Please revert the changes, only add the required routing with the previous ones. To know more about it, you can view PR #661.

Reply:
After reverting I'm getting the previous error

Reply:
Can you upload the screenshot of environment.ts file?

Reply:
![Screenshot from 2020-06-06 16-45-23](https://user-images.githubusercontent.com/36980003/83942868-25ca0e80-a815-11ea-8c81-09b209504f88.png)

Reply:
same. But not working

Reply:
Okay. I am trying in my system.

Till then can you specify the path you are following for loans module.

Reply:
It worked. Thanks :)